app/model/repository/Users.php

```diff
@@ -19,6 +21,77 @@ public function getByEmail(string $email): ?User {
     return $this->findOneBy([ "email" => $email ]);
   }
 
+  // Known order by commands and their translation to Doctrine column names.
+  private static $knownOrderBy = [
+    'name' =>      [ 'u.lastName', 'u.firstName' ],
+    'email' =>     [ 'u.email' ],
+    'createdAt' => [ 'u.createdAt' ],
+  ];
+
+  /**
+   * Fetch users for pagination endpoint (filtered and sorted).
+   * @param Pagination $pagination The object holding pagination metadata.
+   * @param $totalCount Referenced variable, into which the total amount of items is returned.
+   */
+  public function getPaginated(Pagination $pagination, &$totalCount)
+  {
+    $qb = $this->createQueryBuilder('u'); // takes care of softdelete cases
+
+    // Set filters ...
+    if ($pagination->hasFilter("search")) {
+      $search = trim($pagination->getFilter("search"));
+      if (!$search) {
+        throw new InvalidArgumentException("filter", "search query value is empty");
+      }
+
+      $expr = $qb->expr()->orX();
+      foreach (["u.firstName", "u.lastName"] as $column) {
+        $expr->add($qb->expr()->like($column, $qb->expr()->literal('%' . $search . '%')));
```

Comment:
`search` parameter is not splitted, this causes that queries like `Martin Polanka` do not find anything.

Reply:
OK, can do.